src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/SubscriptionsController.cs

```diff
@@ -148,7 +161,16 @@ public StartSubscriptionUpdateWorkItem(IDependencyUpdater dependencyUpdater)
 
             public Task ProcessAsync(JToken argumentToken)
             {
-                return _dependencyUpdater.StartSubscriptionUpdateAsync(argumentToken.Value<Guid>());
+                int buildId = argumentToken["BuildId"].Value<int>();
```

Comment:
`jThing["example"].Value<type>()` is the same as `iThing.Value<int>("example")`. I usually prefer the second (fewer method calls and the square braces) but it's not a huge deal. :-)

Reply:
Applied